README.md

```diff
@@ -1 +1,31 @@
 # ResetPasswordBundle
+
+Worrying about how to deal with users that can't remember their password? We've got you covered! This bundle provides a secure out of the box solution to allow users to reset their forgotten passwords.
+
+## Requirements
+
+Most Symfony application's using Symfony 4.4 or newer should already have all the required dependencies to use this bundle, and the optional traits that are included with it. 
+
+If your application isn't a traditional Symfony web application, see the [detailed requirements](https://github.com/SymfonyCasts/reset-password-bundle/wiki/Dependencies) in our wiki.
+
+## Installation
+
+@TODO - To be added upon release
+
+## Configuration
+
+We highly recommend using `bin/console make:reset-password` to generate and configure the required application files that the bundle needs to function as intended. 
+
+Symfony's Flex creates the bundle's `config/packages/reset_password.yaml` config file. The `reset_request_repository` param is populated with a fake request repository value which must be changed before use. `bin/console make:reset-password` has you covered there too.
```

Comment:
I guess overall there are 2 install & setup paths:

1) with make:reset-password
2) without make:reset-password

Maybe we change this section to `Usage` and sub-divide into two sub-sections for the two things. The first part would be super short: run that command and celebrate. The second section would have a few more details.

Reply:
refactored - 

without make:reset-password; see guide...